native/hiltests/InformationRN4Test.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+ *
+ * Twilio Massive SDK
+ *
+ * Copyright (c) 2019 Twilio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "catch.hpp"
+
+#include <iostream>
+#include <string>
+#include "modem/OwlModemRN4.h"
```

Comment:
This fails to be found when compiled by Arduino IDE.  If compilation in Arduino IDE is not needed/desired, we should wrap this and the other test files in a `#ifndef ARDUINO` block to prevent compilation.  The IDE will grab anything that looks like a source file, regardless of where it lives, and try to compile it otherwise.